### ISSUE TITLE
Building a kernel takes 1.4x longer

### ISSUE THREAD

**Issue by marioroy** [clearlinux/distribution#3107]
CL 41660 (since CL 41570)

Previously CL 41560, building a kernel took 5m15s on the AMD box. Now, 7m30s.
```text
time ./xm-build bore
```

**Comment by marioroy** [comment]
I added the following to the spec file. No improvement. Still 7m30s to build the kernel.

```text
export GCC_IGNORE_WERROR=1
CLEAR_INTERMEDIATE_CFLAGS="$CLEAR_INTERMEDIATE_CFLAGS -fdebug-types-section -femit-struct-debug-baseonly -fno-lto -g1 -gno-column-info -gno-variable-location-views -gz=zstd "
CLEAR_INTERMEDIATE_FCFLAGS="$CLEAR_INTERMEDIATE_FFLAGS -fdebug-types-section -femit-struct-debug-baseonly -fno-lto -g1 -gno-column-info -gno-variable-location-views -gz=zstd "
CLEAR_INTERMEDIATE_FFLAGS="$CLEAR_INTERMEDIATE_FFLAGS -fdebug-types-section -femit-struct-debug-baseonly -fno-lto -g1 -gno-column-info -gno-variable-location-views -gz=zstd "
CLEAR_INTERMEDIATE_CXXFLAGS="$CLEAR_INTERMEDIATE_CXXFLAGS -fdebug-types-section -femit-struct-debug-baseonly -fno-lto -g1 -gno-column-info -gno-variable-location-views -gz=zstd "
CFLAGS="$CLEAR_INTERMEDIATE_CFLAGS"
CXXFLAGS="$CLEAR_INTERMEDIATE_CXXFLAGS"
FFLAGS="$CLEAR_INTERMEDIATE_FFLAGS"
FCFLAGS="$CLEAR_INTERMEDIATE_FCFLAGS"
ASFLAGS="$CLEAR_INTERMEDIATE_ASFLAGS"
LDFLAGS="$CLEAR_INTERMEDIATE_LDFLAGS"
export GOAMD64=v3
```

**Comment by marioroy** [comment]
I decided to cancel the ClearMod project.

**Comment by marioroy** [comment]
This is resolved building the Clear 6.8.10 kernel. Possibly Clear "config" changes comparing 6.8.9 and 6.8.10.

**Comment by marioroy** [comment]
Thank you for resolving the issue. I updated the ClearMod repo and bumped to 6.8.10.
